### ISSUE TITLE
Accessibility : Incorrect focus order on <ebay-infotip/> component

### ISSUE THREAD

**Issue by udaya888** [eBay/ebayui-core#1504]
# Bug Report

## Description
When the overlay open the focus is not on the close (x) button. Once we press the tab the focus is going to inner links instead of the close button. Also, if we press the tab the focus is going away from the overlay.
Please refer this bug : https://jirap.corp.ebay.com/browse/EBAYUI-569

## Screenshots
![image](https://user-images.githubusercontent.com/8204473/131052144-c3590570-afbf-4ae7-8a0e-82ddb182c44c.png)

**Comment by agliga** [comment]
I'm not sure what the issue is.
From what I can see is that after opening infotip, you should be able to tab through it. The first item in the list to tab through should be the close button. 
I tested locally and I saw that it was tabbing through the close button just fine. Since infotip isn't a dialog, it does not change focus to close button, but rather you should be able to tab through to it first. 
One thing to be weary of is to make sure you are opening the infotip only using keyboard. If you open it with mouse and then switch to keyboard the functionality might be different. 
Also just one last comment, since I see your using safari, make sure you have this option enabled on Safari (settings > advanced).
<img width="418" alt="Screen Shot 2021-08-31 at 2 51 18 PM" src="https://user-images.githubusercontent.com/1755269/131580524-e56cf7f4-13db-45c5-b41a-dcaef692e757.png">

**Comment by ianmcburnie** [comment]
@udaya888 Did you read Andrew's comment? Are you still able to repro and rule out anything in your app code that might be doing this?

**Comment by ianmcburnie** [comment]
Neither of these are issues, and are instead "by design".
